### PR TITLE
[PotentialFlow] Adding Perturbation Potential Flow Elements 2 of 3

### PR DESCRIPTION
**Pull request by inigolcanalejo**
This PR introduces the following modifications in the perturbation elements:

- Computation of the velocity
- Computation of the RHS
- Separation of the computation of the RHS and LHS to follow Kratos standards
- Update in the apply far field process to account for perturbation fields
- Corresponding unit tests

### CODE REVIEW

File: applications/CompressiblePotentialFlowApplication/custom_elements/compressible_perturbation_potential_flow_element.cpp [hunk 1]
```diff
@@ -301,13 +309,16 @@ template <int Dim, int NumNodes>
 void CompressiblePerturbationPotentialFlowElement<Dim, NumNodes>::GetEquationIdVectorKuttaElement(
     EquationIdVectorType& rResult) const
 {
+    const auto& r_geometry = this->GetGeometry();
     // Kutta elements have only negative part
     for (unsigned int i = 0; i < NumNodes; i++)
     {
-        if (!GetGeometry()[i].GetValue(TRAILING_EDGE))
-            rResult[i] = GetGeometry()[i].GetDof(VELOCITY_POTENTIAL).EquationId();
+        //r_geometry[i].SetLock();
```
Comment (marcnunezc) [inline applications/CompressiblePotentialFlowApplication/custom_elements/compressible_perturbation_potential_flow_element.cpp]: Commented code here

File: applications/CompressiblePotentialFlowApplication/custom_elements/compressible_perturbation_potential_flow_element.cpp [hunk 2]
```diff
@@ -301,13 +309,16 @@ template <int Dim, int NumNodes>
...
+        if (!r_geometry[i].GetValue(TRAILING_EDGE))
+            rResult[i] = r_geometry[i].GetDof(VELOCITY_POTENTIAL).EquationId();
         else
-            rResult[i] = GetGeometry()[i].GetDof(AUXILIARY_VELOCITY_POTENTIAL).EquationId();
+            rResult[i] = r_geometry[i].GetDof(AUXILIARY_VELOCITY_POTENTIAL).EquationId();
+        //r_geometry[i].UnSetLock();
```
Comment (marcnunezc) [inline applications/CompressiblePotentialFlowApplication/custom_elements/compressible_perturbation_potential_flow_element.cpp]: Here as welll

**Review by marcnunezc — COMMENTED** [review]
Sorry for the delayed review! Minor comment on commented lines, not sure if you want to keep them this way or you just forgot to remove/add them

**Comment by inigolcanalejo** [comment]
Thanks for the review @marcnunezc! I removed the unused commented lines

**Review by marcnunezc — APPROVED** [review]

**Pull request merged**